### DISCUSSION TITLE
Should I version control the "typings" directory or should I git-ignore it?

### DISCUSSION THREAD

**Discussion by danielfalbo** [microsoft/pyright#1612] [Q&A] [answered]
I mean the [type-stubs](https://github.com/microsoft/pyright/blob/master/docs/type-stubs.md) dir

**Comment by erictraut** [comment, marked as answer]
It's your choice — depends on your intent. In most cases, I recommend checking it in to the repo so you have repeatable type checking results, even between developers.
